### PR TITLE
[Notifier] Use mb_strlen() to check the length limit

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 5.x
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]

**Comment by chalasr** [comment]
This should apply on `DiscordTransport` as well?

**Comment by nicolas-grekas** [comment]
> This should apply on DiscordTransport as well?

yes, in 5.2 with #39576

File: src/Symfony/Component/Notifier/Bridge/Slack/Block/SlackHeaderBlock.php [hunk 1]
```diff
@@ -25,7 +25,7 @@ final class SlackHeaderBlock extends AbstractSlackBlock
 
     public function __construct(string $text)
     {
-        if (\strlen($text) > self::TEXT_LIMIT) {
+        if (mb_strlen($text, 'UTF-8') > self::TEXT_LIMIT) {
```
Comment (jderusse) [inline src/Symfony/Component/Notifier/Bridge/Slack/Block/SlackHeaderBlock.php:28]: I thought slack counts in bytes => https://github.com/symfony/symfony/pull/39410/files#r539845790

**Comment by nicolas-grekas** [comment]
> I thought slack counts in bytes => #39410 (files)

thanks for the link, I missed this reply!

**Pull request closed without merging**